SubmittingPatches.rst

```diff
@@ -392,16 +392,19 @@ Title of pull requests and title of commits
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The text up to the first empty line in a commit message is the commit
-title. Ideally it is a single short line less than 50 characters,
+title. Ideally it is a single short line of at most 72 characters,
```

Comment:
Since we're talking of an "ideal" case here, I guess we can still stick to the 50 limit?

Reply:
Yes I believe we can make meaningful title in 50 characters..